gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -9,6 +9,20 @@ module ReactRenderable
     def render_react
       raise DoNotIntegrationTestError if Rails.env.test?
 
+      if Rails.env.development?
+        # Allow concurrent loading to prevent this thread's wait lock from
+        # blocking class loading in controllers called by the Node server.
+        ActiveSupport::Dependencies.interlock.permit_concurrent_loads do
```

Comment:
If you are permitting concurrent loads here why do you still need the monkey patch?

Reply:
This is always safe; no need for an environment check.

Reply:
. We can remove this if/else and always call the `permit_concurrent_loads`